src/containers/ViewerPanel/index.tsx

```diff
@@ -431,6 +462,62 @@ class ViewerPanel extends React.Component<ViewerPanelProps, ViewerPanelState> {
         }
     };
 
+    private updateDisplayBreakpoints = () => {
+        if (location.pathname === EMBED_PATHNAME) {
```

Comment:
to me it makes more sense to just not call this function than to have this here. since the pattern set up above by the code is to have the checks there. Otherwise, looking at it I would assume you're already updating the breakpoints on mount